metadata.json

```diff
@@ -8,8 +8,8 @@
   "project_page": "https://github.com/puppetlabs/puppetlabs-apache",
   "issues_url": "https://tickets.puppetlabs.com/browse/MODULES",
   "dependencies": [
-    {"name":"puppetlabs/stdlib","version_requirement":">= 4.2.0 < 5.0.0"},
-    {"name":"puppetlabs/concat","version_requirement":">= 1.1.1 < 3.0.0"}
+    {"name":"puppetlabs/stdlib","version_requirement":">= 4.13.1 < 5.0.0"},
+    {"name":"puppetlabs/concat","version_requirement":">= 2.2.1 < 3.0.0"}
```

Comment:
I meant concat. It's now pinned to ```< 3.0.0``` but I think it can be ```< 5.0.0``` so it can use Puppet 4 types there as well.